jenkins/databricks/build.sh

```diff
@@ -52,13 +52,7 @@ declare -A artifacts
 initialize()
 {
     # install rsync to be used for copying onto the databricks nodes
-    sudo apt install -y rsync
-
-    if [[ ! -d $HOME/apache-maven-3.6.3 ]]; then
-        wget https://archive.apache.org/dist/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz -P /tmp
-        tar xf /tmp/apache-maven-3.6.3-bin.tar.gz -C $HOME
-        sudo ln -s $HOME/apache-maven-3.6.3/bin/mvn /usr/local/bin/mvn
-    fi
+    sudo apt install -y maven rsync
```

Comment:
We should not revert this part, deterministic Maven version is more robust either way regardless whether the apt package for Maven is back